.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Please fill out relevant sections below; remove those that are unused.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Platform information:**
+ - Fritz version:  (find this in the [About](https://fritz.science/about) page)
+ - Browser
+   - [ ] I'm browsing on a desktop
+   - [ ] I'm browsing on my phone
+
```

Comment:
- could also have tablet as an option (with phone and computer these are the three main options)
- bug could be API related so maybe want to allow this to.

Reply:
Fixed.